pkg/filters/containers.go

```diff
@@ -84,11 +58,3 @@ func (filter *ContainerFilter) InitBPF(bpfModule *bpf.Module, conts *containers.
 
 	return nil
 }
-
-func (filter *ContainerFilter) FilterOut() bool {
```

Comment:
I don't see that you've updated references to this function, does this compile?

Reply:
ditto, embedded struct inherits this from StringFilter, and ditto that I change this filter later